### PR TITLE
fix(example): wizard annotation and one back link

### PR DESCRIPTION
**Pull request by gdi2290**
fix: router di
fix: title
fix: back button on first step
feat: show state at the end

sidenote: It's not very clear that you're using bindToController although I understand the reason behind it

### CODE REVIEW

**Comment by gdi2290** [comment]
I just noticed I have an old copy gg. one sec

**Comment by lfryc** [comment]
:+1: fix works perfectly

File: examples/angular-1/wizard/components/three/three.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <section>
-  <h2>{{three.question}}</h2>
+  <h2>{{ three.question }}</h2>
```
Comment (lfryc) [inline examples/angular-1/wizard/components/three/three.html]: my $0.02: for a reason of consistency with other examples, I'd leave it without extra spaces

**Comment by btford** [comment]
Landed as e9a6e26cfe870e58a75eafa71f1f8567a7960c31. Thanks!

**Pull request closed without merging**